bodhi/server/templates/override.html

```diff
@@ -138,6 +138,7 @@ <h2 class="pull-left">New Override</span>
 
     <input type="hidden" name="csrf_token" value="${request.session.get_csrf_token()}"/>
 
+    %if request.user and 'mandatory_packager_groups' in request.user.groups:
```

Comment:
This does not appear to work in my development checkout. I am logged in and I am in the correct group, but I am not shown these buttons.